content/guides/pricing-concepts.md

```diff
@@ -0,0 +1,120 @@
+
+The way LOS records' `rates` work is the following: each consecutive rate is the price for the given length of stay (hence Length Of Stay Records). For a one-day stay, the price will be equal to the first rate, for a two-day stay, the price will be equal to the second rate and so forth.
+
+There are three `kind`s of LOS records, but two of them are particularly important:
+
+* `rental_price` - the LOS records will contain the rates for booking's `initial_price` (check Live Quote docs for reference).
+* `final_price ` - the LOS records will contain the rates for booking's `final_price`, which means it covers excluded taxes and required fees as well (check Live Quote docs for reference).
+
+### 3. Rates and rates rules (not recommended).
+
+Besides LOS records, there is one more form of a distributed calculations cache - [rates](http://developers.bookingsync.com/reference/endpoints/rates/). They are in some sense similar to LOS records, but the rates are divided into the periods without a reference to the specific day for which they were calculated. That means that not all [rates rules](http://developers.bookingsync.com/reference/endpoints/rates_rules/) could have been applied when generating them (like `late_booking` or `early_booking`). If you **really** have to use rates, you should apply rates rules that make sense on top of them when calculating the price for a given booking, but even then the price most likely will not be the same as from LOS records or Live Quote.
+
+### 4. Nightly rates maps, seasons, periods and rates rules (not recommended and not supposed to be used for exact price).
+
+It is **absolutely** not recommended to use directly any of those to calculate the price of the booking; nevertheless, it might give you some insight how things work on a lower-level.
+
+Seasons and periods are resources which modify rentals' base rate for given dates range. That way, you can get the exact "raw" nightly rate for the rental for a given day (which means without any rates rules applied). [Nightly rates map](http://developers.bookingsync.com/reference/endpoints/nightly_rate_maps/) is the equivalent resource to combined seasons and periods; the format is just more compact and much easier to manage outside the BookingSync, via a third-party application.
+
+[`Rates`](http://developers.bookingsync.com/reference/endpoints/rates/) are calculated based on the seasons/periods/nightly rates maps, so those resources should not be used for any calculation, they should be considered as an implementation detail.
```

Comment:
Confused about this description, not inline with the above details.

Also good to describe rates rules however

Reply:
How about describing rates rules in a separate PR to merge this one possibly fast?